rocAL/rocAL/include/api/rocal_api_augmentation.h

```diff
@@ -24,76 +24,100 @@ THE SOFTWARE.
 #define MIVISIONX_ROCAL_API_AUGMENTATION_H
 #include "rocal_api_types.h"
 
-/// Accepts U8 and RGB24 input.
-// Rearranges the order of the frames in the sequences with respect to new_order.
-// new_order can have values in the range [0, sequence_length).
-// Frames can be repeated or dropped in the new_order.
-/// \param context
-/// \param input
-/// \param new_order
-/// \param is_output
-/// \return
-extern "C" RocalTensor  ROCAL_API_CALL rocalSequenceRearrange(RocalContext p_context, RocalTensor p_input,
+/*!
+ * \brief Rearranges the order of the frames in the sequences with respect to new_order.
+ * Frames can be repeated or dropped in the new_order.
+ * \ingroup group_rocal_augmentations
+ * \note: Accepts U8 and RGB24 input.
+ * \param context context for the pipeline.
```

Comment:
Add the correct arg names here p_context & p_input
Change the explanations to tell that its a pointer variable